test/test_helper.rb

```diff
@@ -1,5 +1,16 @@
 require 'simplecov'
-SimpleCov.start 'rails'
+require 'simplecov-lcov'
+
+SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
+SIMPLECOV_FORMATTERS = [
```

Comment:
Style/MutableConstant: Freeze mutable objects assigned to constants.

Reply:
Fixed.